electronics_model/PriceGetter.py

```diff
@@ -0,0 +1,69 @@
+from typing import List, Tuple, Dict
+
+import edgir
+from edg_core import BaseBackend, CompiledDesign, TransformUtil
+import os
+import csv
+import re
+
+
+class GetPrice:
+    def run(lcsc_part_number: str, quantity: int) -> float:
+        cur_path = os.path.dirname(__file__)
+        parts_library = os.path.relpath(
+            '..\\electronics_lib\\resources\\Pruned_JLCPCB SMT Parts Library(20220419).csv', cur_path)
+
+        price_string: str = 'NA'
```

Comment:
```suggestion
        price_string: Optional[str] = None
```
While this will work, consider using the Optional type instead so it's strongly typed (the missing-value type is different) and the type system can help make sure all cases are considered.

Reply:
got it, fixed